ribs/archives/_archive_base.py

```diff
@@ -152,22 +146,22 @@ def __init__(self, solution_dim, cells, behavior_dim, seed=None, dtype=np.float6
         self._num_occupied = 0
         self._occupied = np.zeros(self._cells, dtype=bool)
         self._occupied_indices = np.empty(self._cells, dtype=np.int32)
-        
+
         self._solutions = np.empty((self._cells, solution_dim),
                                    dtype=self.dtype)
         self._objective_values = np.empty(self._cells, dtype=self.dtype)
         self._behavior_values = np.empty((self._cells, self._behavior_dim),
```

Comment:
change to `self._measures_batch`

Reply:
Will change this later on. I think this should not be batch since it is the archive storage, so maybe `measures_storage`